pkg/cli/cmds/server.go

```diff
@@ -60,6 +60,7 @@ type Server struct {
 	DatastoreCAFile          string
 	DatastoreCertFile        string
 	DatastoreKeyFile         string
+	DatastoreTLS             bool
```

Comment:
This PR continues to conflate TLS configuration for kine, and TLS for the the datastore behind kine. Don't do that.

The existing `DatastoreXXX` fields all set config for the kine's connection to the backend datastore (the database server). This flag is for enabling TLS on kine itself; it should be named appropriately.
```suggestion
	KineTLS             bool
```